### PR TITLE
Flexible watch command

### PR DESCRIPTION
**Pull request by machinemessiah**
_I couldn't find a way to run a single watch command which would watch and update extensions, apps, and styles/css altogether, so I was running separate commands in separate terminal instances. This is an amateur attempt to allow any/all of these updates with a single watch command in a single terminal instance. If this functionality is already incorporated, just ignore this (a comment letting me know HOW to do it if it is already possible would be much appreciated!)._

I've modified spotify.go so any or all of [extensions, style/css, apps] can be live-updated using a single watch command by just adding the appropriate letter(s) after a single "-" like spicetify watch -(any combination of **e**, **a**, and **l** or **s**).

I changed the -e and -a watch args to enable live-updating by default also, though if this isn't a good idea it's a simple change to have only -l enable the live-updating feature.

**I need to stress that I have NO CLUE what I'm doing when it comes to go. I don't have any experience with the language, so these changes, while they work for me, are likely horribly inefficient and badly implemented.**

### CODE REVIEW

**Comment by afonsojramos** [comment]
I still haven't had time to properly test it out, I'll try to take a closer look on the weekend.

**Review by afonsojramos — CHANGES_REQUESTED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+* text=auto
```
[the remark this answers is not in the document]
Comment (machinemessiah) [reply .gitattributes]: Totally forgot I'd added that file when I first made the branch, sorry @afonsojramos !

**Review by itsmeow — APPROVED** [review]

**Review by afonsojramos — CHANGES_REQUESTED** [review]
Looking good, but can you install the go formatter? Because in our codebase we don't have semicolons.

**Comment by machinemessiah** [comment]
> Looking good, but can you install the go formatter? Because in our codebase we don't have semicolons.

Messed up when I committed changes in response to this, so now the automatic prettier check fails. Tried to fix my mistake but failed; now I honestly don't know what to do.
Please advise (I'm useless with git sorry...).

**Review by afonsojramos — APPROVED** [review]
Awesome!! Thanks @machinemessiah!

**Comment by afonsojramos** [comment]
> In regards to the trailing semicolons in the go codebase, is there a reason this isn't something the current formatting config could be modified to take care of? Obviously the CLRF/LF mismatch and the trailing semicolons formatting issues which have cropped up here are both things on my end, but wouldn't it be simpler to just make sure those sort of issues are corrected automatically (since they're stylistic choices rather than mistakes which break any logic)?

Regarding this, if you install the recommended Go tools in vscode it automatically installs the formatter, so I'm not sure this is needed. 

Additionally, I think that the industry overall is kind of against applying lint fixes in the CI, and more in favor of failing checks. However, it is true that we do not have one running for Golang. I will add an issue reporting this!

**Pull request merged**